src/app/app.module.ts

```diff
@@ -42,7 +41,7 @@ import {LogOutKeepAliveService} from './services/keep-alive/keep-alive.service';
 import { EnvironmentConfig, ENVIRONMENT_CONFIG } from '../models/environmentConfig.model'
 
 export const metaReducers: MetaReducer<any>[] = !config.production
-  ? [storeFreeze]
+  ? []
```

Comment:
@ronaldmansveld - is there any point having `!config.production ? [] : []`, given the outcome of the ternary is the same in both cases?

Reply:
Updated. Left the declaration as it's exported.